packages/suite/src/components/suite/CoinList/CoinList.tsx

```diff
@@ -75,9 +75,7 @@ export const CoinList = ({
                     getCoinUnavailabilityMessage(unavailableReason);
                 const tooltipString = discoveryTooltip || lockedTooltip || unavailabilityTooltip;
 
-                const coinLabel = blockchain[symbol].backends.selected
-                    ? 'TR_CUSTOM_BACKEND'
-                    : (label as TranslationKey);
+                const label = getCoinLabel(features, isTestnet, !!blockchain[symbol].backends.selected);
```

Comment:
nitpick: It might be more readable with arguments as an object. Especially because of last argument: `hasCustomBackend: !!blockchain[symbol].backends.selected`